lib/stimulus_reflex/reflex_data.rb

```diff
@@ -4,7 +4,7 @@ class StimulusReflex::ReflexData
   attr_reader :data
 
   def initialize(data)
-    @data = data
+    @data = data.deep_transform_keys { |k| k.to_s.underscore }.with_indifferent_access
```

Comment:
Hm, what is the motivation behind making this with indifferent access?

Reply:
I think it's generally nicer and more idiomatic to work with snake case symbols rather than camel case strings in Ruby.

https://github.com/rubocop/ruby-style-guide?tab=readme-ov-file#snake-case-symbols-methods-vars
https://github.com/rubocop/ruby-style-guide?tab=readme-ov-file#symbols-as-keys